.travis.yml

```diff
@@ -4,7 +4,20 @@ jdk:
   - oraclejdk7
   - openjdk6
   - oraclejdk8
+env:
```

Comment:
Yes. Here is an example you can take a look at 

https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix